pkg/generator/ingress_translator_test.go

```diff
@@ -970,4 +1175,45 @@ var (
 			"tls.key": privateKey,
 		},
 	}
+	caSecret = &corev1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: "knative-testing",
+			Name:      "test-ca",
+		},
+		Data: map[string][]byte{
+			caDataName: cert,
+		},
+	}
 )
+
+func typedConfig(http2 bool) *envoy_core_v3.TransportSocket_TypedConfig {
+	alpn := []string{""}
+	if http2 {
+		alpn = []string{"h2"}
+	}
+	tlsAny, _ := anypb.New(&auth.UpstreamTlsContext{
+		CommonTlsContext: &auth.CommonTlsContext{
+			AlpnProtocols: alpn,
+			TlsParams: &auth.TlsParameters{
+				TlsMinimumProtocolVersion: auth.TlsParameters_TLSv1_2,
+			},
+			ValidationContextType: &auth.CommonTlsContext_ValidationContext{
+				ValidationContext: &auth.CertificateValidationContext{
+					TrustedCa: &envoy_core_v3.DataSource{
+						Specifier: &envoy_core_v3.DataSource_InlineBytes{
+							InlineBytes: cert,
+						},
+					},
+					MatchSubjectAltNames: []*envoy_matcher_v3.StringMatcher{{
+						MatchPattern: &envoy_matcher_v3.StringMatcher_Exact{
+							Exact: "test-san",
```

Comment:
You can refactor `ActivatorSAN: "test-san"` as a constant and use it in all occurrences. Same for `test-ca`

Reply:
There are only two places so far, so I think it is fine without using the constant.